web/flat-web/src/components/Tips/ErrorTips.tsx

```diff
@@ -9,8 +9,8 @@ export const errorTips = (e: unknown): void => {
     }
 
     if (e instanceof ServerRequestError) {
-        void message.error(i18n.t(e.errorMessage));
+        void message.error({ content: i18n.t(e.errorMessage), key: i18n.t(e.errorMessage) });
```

Comment:
@BlackHole1 I used the message itself as the key because I believe it's possible to have different messages generated for the same error code (some are dynamically generated.) You'd know better if that's relevant to the repo or not. If most of the errors being thrown have simple content like 'Network Error' instead of something more complicated (for example something that includes the URL of a resource), it might not be as relevant as I assumed, but I thought I'd at least mention it.

Reply:
You have a point.